CRM/Member/Import/Parser/Membership.php

```diff
@@ -29,22 +29,15 @@
  *
  * @package CRM
  * @copyright CiviCRM LLC (c) 2004-2019
- * $Id$
- *
  */
 
-
-require_once 'api/api.php';
```

Comment:
@eileenmcnaughton i'm wondering if this was a APIv2 issue or something

Reply:
it's certainly historical! That file is always included now - although deprecatedUtils - if used - maybe not